packages/core/src/services/essentials/dom-utils.ts

```diff
@@ -137,14 +137,16 @@ export class DOMUtils extends Service {
 		return finalDimensions;
 	}
 
-	static appendOrSelect(parent, query) {
+	static appendOrSelect(parent, query, id?) {
```

Comment:
a bit weird this new parameter. It's inconsistent with the rest of the parameters and the way this function works